src/schemas/40-10007/schema.js

```diff
@@ -310,6 +315,9 @@ let schema = {
             properties: {
               attachmentId: {
```

Comment:
Any interest in enumerating the enum and enumNames here?
```
                          'enum': [
                            '1',
                            '2',
                            '3',
                            // '4',
                            '5',
                            '6'
                          ],
                          enumNames: [
                            'Discharge',
                            'Marriage related',
                            'Dependent related',
                            // 'VA preneed form',
                            'Letter',
                            'Other'
```

Reply:
Done.

Reply:
could you clarify what you mean here @cehsu?

Reply:
The `attachmentId` was just defined as a `string`.
On the form it's a dropdown, so I added the `enum` definition as @cehsu defined it in her comment.

Reply:
haha, I was looking at your code that had already implemented her feedback so I was confused :)